website/source/docs/providers/aws/r/elasticache_replication_group.html.markdown

```diff
@@ -0,0 +1,71 @@
+---
+layout: "aws"
+page_title: "AWS: aws_elasticache_replication_group"
+sidebar_current: "docs-aws-resource-elasticache-replication-group"
+description: |-
+  Provides an ElastiCache Replication Group resource.
+---
+
+# aws\_elasticache\_replication\_group
+
+Provides an ElastiCache Replication Group resource.
+
+~> **Note:** We currently do not support passing a `primary_cluster_id` in order to create the Replication Group.
+
+## Example Usage
+
+```
+resource "aws_elasticache_replication_group" "bar" {
+  replication_group_id          = "tf-replication-group-1"
+  replication_group_description = "test description"
+  node_type                     = "cache.m1.small"
+  number_cache_clusters         = 2
+  engine                        = "redis"
+  port                          = 6379
+  parameter_group_name          = "default.redis2.8"
+  availability_zones            = ["us-west-2a", "us-west-2b"]
+  automatic_failover_enabled    = true
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `replication_group_id` – (Required) The replication group identifier. This parameter is stored as a lowercase string.
+* `replication_group_description` – (Required) A user-created description for the replication group.
+* `number_cache_clusters` - (Required) The number of cache clusters this replication group will initially have.
+ If Multi-AZ is enabled , the value of this parameter must be at least 2. Changing this number will force a new resource
+* `node_type` - (Required) The compute and memory capacity of the nodes in the node group.
+* `engine` - (Required) The name of the cache engine to be used for the cache clusters in this replication group. The only valid value is Redis.
```

Comment:
Should we even have this then, if the only value accepted is `redis`? Seems like needless work for our users